tasklist/importer-860/src/main/java/io/camunda/tasklist/zeebeimport/v860/processors/es/VariableZeebeRecordProcessorElasticSearch.java

```diff
@@ -102,4 +109,120 @@ private UpdateRequest getVariableQuery(final VariableEntity entity) throws Persi
           e);
     }
   }
+
+  private UpdateRequest persistVariableToListView(
+      final Record record, final VariableRecordValueImpl recordValue) throws PersistenceException {
+    final VariableEntity variableEntity = getVariableEntity(record, recordValue);
+    VariableListViewEntity variableListViewEntity = createVariableInputToListView(variableEntity);
+
+    if (isTaskOrSubProcessVariable(variableEntity)) {
+      variableListViewEntity = associateVariableWithTask(variableListViewEntity);
+      return prepareUpdateRequest(variableListViewEntity, variableListViewEntity.getScopeKey());
+    } else if (isProcessScope(variableEntity)) {
+      variableListViewEntity = associateVariableWithProcess(variableEntity, variableListViewEntity);
+      return prepareUpdateRequest(variableListViewEntity, variableEntity.getProcessInstanceId());
```

Comment:
in this case variableListViewEntity.getScopeKey() ==  variableEntity.getProcessInstanceId() , isn't?
so `variableListViewEntity.getScopeKey()` can be used in both cases